### PR TITLE
Update info.html

### PR DESCRIPTION
**Pull request by VinLi04**

### CODE REVIEW

File: info.html [hunk 1]
```diff
@@ -1,14 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <link href="style.css" rel="stylesheet">
```
Comment (Yassin-Kortam2) [inline info.html:4]: This is needed for the css

File: info.html [hunk 2]
```diff
@@ -1,14 +1,60 @@
...
+    <link href="style.css" rel="stylesheet" type="text/css">
   </head>
-<body>
- <div id="navbar">
-  <a class="active" href="info.html">Info</a>
-  <a href="index.html" style="float:right">Home</a>
-  <a href="about-us.html" style="float:right">About Us</a>
-</div>
+
+    <body class="info">
```
Comment (Yassin-Kortam2) [inline info.html:8]: its better as an ID than a class because this is only one page

File: info.html [hunk 3]
```diff
@@ -1,14 +1,60 @@
...
+      <h2 style="text-align:center"><a style="color:blue";href="https://www.evc.edu/academics/special-academic-programs/accel-middle-college">Visit ACCEL Middle College!</a></h2>
+      <p>
+        Accel Middle College Program is a cooperative program between East Side Union High School District and Evergreen Valley Community College. Accel juniors and seniors simultaneously attend college courses during the morning and high school classes in the afternoon. Student receive high school and college credit, which enables them to complete their high school education while experiencing college early. All classes, including high school courses, are held on the Evergreen Valley College campus.
+      </p>
+      </div>
+
+      </body>
+    </html>
   <script>
 window.onscroll = function() {myFunction()};
```
Comment (Yassin-Kortam2) [inline info.html:59]: script tag should be inside the body tag

**Review by Yassin-Kortam2 — CHANGES_REQUESTED** [review]
The instructions should be on a pdf document that is linked to on the info page, not directly on it. Also, please clean up the layout. Paragraphs have no padding, the font is still the default, links need to be stylized better.

File: info.html (hunk 2 above)
Comment (ramaduraianirudh9867) [reply info.html:8]: @kellylin118 @VinLi04 @aparna2003

File: info.html (hunk 3 above)
Comment (ramaduraianirudh9867) [reply info.html:59]: @kellylin118 @VinLi04 @aparna2003

**Comment by ramaduraianirudh9867** [comment]
@kellylin118 @VinLi04 @aparna2003 please see the comments above, and fix as appropriate.

**Pull request closed without merging**